docs/docs-content/automation/palette-sdk/palette-sdk.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_label: "Palette Go SDK"
+title: "Palette Go SDK"
+description: "Learn how to get started with the Palette Go SDK."
+hide_table_of_contents: false
+tags: ["palette-sdk"]
+---
+
+The Palette Go Software Developer Toolkit (SDK) allows Go developers to interact with several public Palette APIs
```

Comment:
suggestion: The Palette Go Software Developer Toolkit (SDK) provides Go developers a user-friendly client for interacting with Palette APIs.

I don't think we should mention "public" APIs because that implies that we have undocumented, private APIs.